mscore/plugin/api/elements.cpp

```diff
@@ -69,6 +69,42 @@ void Note::setTpc(int val)
             set(Pid::TPC2, val);
       }
 
+
+//---------------------------------------------------------
+//   Chord::add
+//---------------------------------------------------------
+
+void Chord::add(Ms::PluginAPI::Element* wrapped)
+      {
+      Ms::Element* s = wrapped->element();
+      if (s)
+            {
+            // Score now owns the object.
+            wrapped->setOwnership(Ownership::SCORE);
```

Comment:
It would probably be good to ensure that the element was previously owned by a plugin. I see now that the same applies also to `Cursor::add()`, I will need to correct it there too.

Reply:
I understand adding an assert for sanity check but what would a retail build do if this isn't the case?

Reply:
I've added the assert but the above question remains.

Reply:
Not an assert, just refuse doing this operation if this happens, possibly with some warning, like
```
if (wrapped->ownership() == Ownership::SCORE) {
      qWarning("Chord::add: cannot add an element: element is already added to a score");
      return;
      }
```
Such kind of errors indicates some logical errors in a plugin, not in MuseScore itself.

Reply:
Makes sense. I had it backwards.
Done.